src/App.js

```diff
@@ -1,12 +1,53 @@
 import React, { Component } from 'react';
+
+import HeroData from "./utils/heroes.json";
+
 import HeroCard from './components/HeroCard';
 import './App.scss';
 
+
+const arrayToMap = (array) => {
+  if (!Array.isArray(array)) {
+      return {};
+  }
+
+  return array.reduce((aggr, obj) => {
+      return {
+          ...aggr,
+          [obj.id]: obj
+      };
+  }, {});
+};
+
+//Render our hero cards for the app to use. 
+const heroRender = (heroList) => {
+
+  return (
+    Object.keys(heroList).map((key) => <HeroCard key={heroList[key].id} heroData={heroList} hero={heroList[key]} counters={heroList[key].counters}/>)
+  )
+}
+
+
 class App extends Component {
+  state = {
+    "hero": null,
+  }
+
+  componentWillMount() {
+    console.log(arrayToMap(HeroData))
+    this.setState({
+        "hero": arrayToMap(HeroData),
+    })
+
+    
+}
+
   render() {
+    
+    let { hero } = this.state;
```

Comment:
Can you clarify why you chose to use a `let` statement over a `const`?

Reply:
And you destructured it, then used `this.state.hero` anyway. ;)

Reply:
I used let because I've generally felt more comfortable using let instead of const. I've never had scope issues, but it feels more "right" to me. That being said, I should be more consistent, since I see that I mix let and const. I also know that const is the standard for everything unless we have to change it later. I'll make the change :) 

Consistency is key!

Reply:
Also, not using the deconstruction was an accident. I glanced over it!